### PR TITLE
Add link to Chinese translation

### PR DESCRIPTION
**Pull request by aakloxu**
I have been translating RBE into [Chinese](https://github.com/rust-lang-cn/rust-by-example-cn) since March, 2016. I've already translated as far as chapter 8 (Functions) and I will keep updated with the English version. I appreciate you reviewing the PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,17 +97,18 @@ The Makefile provides the following recipes:
   under `localhost:4000`
 * `make test`: will check all the rust source files for compilation errors
 
+## Translation to another languages
```
Comment (steveklabnik) [inline README.md]: Could you make this "Translations to other languages" languages, please?

**Comment by steveklabnik** [comment]
Thanks so much! A small grammar nit, but I'm glad to see more translations :)

File: README.md (hunk 1 above)
Comment (aakloxu) [inline README.md]: Yes. I will update it soon.

**Comment by steveklabnik** [comment]
Thank you so much!

**Pull request merged**